tests/test_marathon.py

```diff
@@ -80,6 +80,63 @@ def test_remove_pod_propagates_DCOSException():
     assert str(exception_info.value) == 'BOOM!'
 
 
+def test_show_pod_builds_rpc_correctly_1():
+    marathon_client, rpc_client = _create_fixtures()
+    marathon_client.show_pod('foo')
+    rpc_client.http_req.assert_called_with(http.get, 'v2/pods/foo')
+
+
+def test_show_pod_builds_rpc_correctly_2():
+    marathon_client, rpc_client = _create_fixtures()
+    marathon_client.show_pod('bar')
+    rpc_client.http_req.assert_called_with(http.get, 'v2/pods/bar')
```

Comment:
Yeah, it's the leftovers of TDD that I didn't refactor. I'll merge a few of the similar test cases together.